src/github_review_prompts/utils/smart_comment_filter.py

```
@@ -170,28 +171,41 @@ def should_create_task(
-        has_technical_indicator = any(
-            indicator in comment_body for indicator in technical_indicators
+        has_actionable_indicator = any(
+            indicator in comment_body for indicator in actionable_indicators
+        )
+        has_informational_indicator = any(
+            indicator in comment_body for indicator in informational_indicators
         )
-        if has_technical_indicator:
+
+        if has_actionable_indicator or has_informational_indicator:
             self.logger.debug(f"技術的指摘マーカー検出 - 除外パターンをスキップ")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)